src/setup/scss/_reset.scss

```diff
@@ -1,4 +1,4 @@
-@import '~reset-css/reset.css';
+@import '~the-new-css-reset/css/reset.css';
```

Comment:
todo: `@import` should not be used anymore. Can you check if this can be replaced with `@use`?

@see https://sass-lang.com/documentation/at-rules/use

Reply:
Throws an error when using @use.
I took a look at the sass documentation, it seems to me, @import is the correct way for this case.

In short:
@use -> The @use rule loads mixins, functions, and variables from other Sass stylesheets
@import -> Sass extends CSS's with the ability to import Sass and CSS stylesheets